modules/standard/IO.chpl

```diff
@@ -1602,6 +1602,11 @@ private extern proc qio_channel_path_offset(threadsafe:c_int, ch:qio_channel_ptr
 private extern proc qio_channel_retain(ch:qio_channel_ptr_t);
 private extern proc qio_channel_release(ch:qio_channel_ptr_t);
 
+private extern proc qio_channel_clear_error(ch:qio_channel_ptr_t);
+private extern proc _qio_channel_set_error_unlocked(ch:qio_channel_ptr_t, err:syserr);
```

Comment:
Oh okay.  It just stood out to me because it was different than everything around it